third_party/2and3/yaml/__init__.pyi

```diff
@@ -22,10 +22,27 @@ def safe_load_all(stream: Union[str, IO[str]]) -> Iterator[Any]: ...
 def emit(events, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=...): ...
 def serialize_all(nodes, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...): ...
 def serialize(node, stream=..., Dumper=..., **kwds): ...
-def dump_all(documents: Sequence[Any], stream: IO[str]=..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> Any: ...
-def dump(data: Any, stream: IO[str]=..., Dumper=..., **kwds) -> Any: ...
-def safe_dump_all(documents: Sequence[Any], stream: IO[str]=..., **kwds) -> Any: ...
-def safe_dump(data: Any, stream: IO[str]=..., **kwds) -> Any: ...
+
+@overload
+def dump_all(documents: Sequence[Any], stream: None=..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> AnyStr: ...
+@overload
+def dump_all(documents: Sequence[Any], stream: IO[str]=..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> None: ...
+
+@overload
+def dump(data: Any, stream: None=..., Dumper=..., **kwds) -> AnyStr: ...
+@overload
+def dump(data: Any, stream: IO[str]=..., Dumper=..., **kwds) -> None: ...
```

Comment:
I think you don't need `=...` in the second overload here and in other functions, otherwise there is clearly an overlap, because `dump(something)` matches both.

Reply:
Yes, that's it: After removing the `=...` it seems to work. Will --amend and push --force